css/style.css

```diff
@@ -0,0 +1,74 @@
+html
+{
+    text-align: center;
+    background-color: ghostwhite;
+ 
+}
+
+.log-in
+{
+    background: white;
+    border: 2px solid gray;
+    border-radius: 3px;
+    padding: 6px 24px;
+    font-size: 15px;
+}
+#sing-up:hover{
```

Comment:
Generally, a lot of inconsistencies in formatting that would be fixed by using prettier, like having { brackets on the same line or on different lines with the classes in some cases and a lack of lines between the classes.

Reply:
I would like to hear more of the details

Reply:
https://prettier.io/ here is a link